### DISCUSSION TITLE
SELF_SIGNED_CERT_IN_CHAIN error  when building the application

### DISCUSSION THREAD

**Discussion by giannineves** [saleor/saleor#9727] [General] [answered]
Hi guys, 

Need help to fix this problem. I already try to run in WSL ALPINE and UBUNTU, always having the exact same error. Below, is the error:
Thanks,

db uses an image, skipping
redis uses an image, skipping
jaeger uses an image, skipping
mailhog uses an image, skipping
Building dashboard
Step 1/13 : FROM node:14
 ---> 531acaaca10c
Step 2/13 : WORKDIR /app
 ---> Using cache
 ---> acc84ffada05
Step 3/13 : COPY package*.json ./
 ---> Using cache
 ---> 5df99038329b
 
**Step 4/13 : RUN npm install
 ---> Running in 32dd5dc71744

> @sentry/cli@1.63.2 install /app/node_modules/@sentry/cli
> node scripts/install.js

info sentry-cli Downloading from https://downloads.sentry-cdn.com/sentry-cli/1.63.2/sentry-cli-Linux-x86_64
Error: Unable to download sentry-cli binary from https://downloads.sentry-cdn.com/sentry-cli/1.63.2/sentry-cli-Linux-x86_64.
Error code: SELF_SIGNED_CERT_IN_CHAIN
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: fsevents@2.3.2 (node_modules/rollup/node_modules/fsevents):
npm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for fsevents@2.3.2: wanted {"os":"darwin","arch":"any"} (current: {"os":"linux","arch":"x64"})
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: fsevents@1.2.13 (node_modules/fsevents):
npm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for fsevents@1.2.13: wanted {"os":"darwin","arch":"any"} (current: {"os":"linux","arch":"x64"})
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: fsevents@2.3.2 (node_modules/chokidar/node_modules/fsevents):
npm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for fsevents@2.3.2: wanted {"os":"darwin","arch":"any"} (current: {"os":"linux","arch":"x64"})
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: fsevents@2.3.2 (node_modules/@graphql-codegen/cli/node_modules/fsevents):
npm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for fsevents@2.3.2: wanted {"os":"darwin","arch":"any"} (current: {"os":"linux","arch":"x64"})

npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! @sentry/cli@1.63.2 install: `node scripts/install.js`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the @sentry/cli@1.63.2 install script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /root/.npm/_logs/2022-05-09T17_22_11_512Z-debug.log
ERROR: Service 'dashboard' failed to build: The command '/bin/sh -c npm install' returned a non-zero code: 1**

**Comment by giannineves** [comment, marked as answer]
Fixed guys.
I changed the firewall rule and everything works fine.

Thanks

**Reply by leador** [reply]
Same error on Ubuntu, can you explain detailed? How to fix this error?

**Reply by giannineves** [reply]
You need to remove the SSL inspection from your firewall. Here, my network admin creates a special rule for my computer, and everything works.
